src/django/api/reports/percent_data_by_source_type.sql

```diff
@@ -9,9 +12,8 @@ FROM (
     FROM api_facilitymatch m
         JOIN api_facilitylistitem i on m.facility_list_item_id = i.id
         JOIN api_source s on i.source_id = s.id
-        JOIN api_contributor c ON s.contributor_id = c.id
```

Comment:
This join was unnecessary, because we only used the Contributor id in the grouping query, and don't need any additional contributor details.